apps/fly-my-rockets/src/app/rockets/components/photos/photos.component.html

```diff
@@ -1,12 +1,18 @@
 <ng-container *ngIf="rocket$ | async as rocket; else loadingSpinner">
   <div class="grid--photos">
     <ng-container *ngFor="let photo of photoUrls; index as index">
-      <img *ngIf="photo.url | async as url"
-           [src]="url"
-           [alt]="'Photo ' + index + 1"
-           class="grid--photo"
-           (click)="showPhoto(photo)"
-      >
+      <mat-card class="grid--photo" *ngIf="photo.url | async as url">
+        <img mat-card-image
+             [src]="url"
+             [alt]="'Photo ' + index + 1"
+        >
+        <mat-menu #photoOptions="matMenu">
```

Comment:
Odd spot for the menu but can't be below mat-card-actions or it throws off a CSS selector looking for the last child. Doesn't seem to affect anything here.